grunt/config/browserify.js

```diff
@@ -24,8 +23,9 @@ var LICENSE_TEMPLATE =
  * All rights reserved.\n\
  *\n\
  * This source code is licensed under the BSD-style license found in the\n\
- * LICENSE file in the root directory of this source tree. An additional grant\n\
- * of patent rights can be found in the PATENTS file in the same directory.\n\
+ * LICENSE file in the root directory of this source tree. An additional\n\
+ * grant of patent rights can be found in the PATENTS file in the same\n\
+ * directory.\n\
```

Comment:
Can you revert these changes. This is intended (to match the formatting of the license as it appears in regular source files).